Nodejs/Product/Nodejs/Intellisense/VsProjectAnalyzer.cs

```diff
@@ -150,12 +150,16 @@ sealed partial class VsProjectAnalyzer : IDisposable {
 
         private void CreateNewAnalyzer(AnalysisLimits limits) {
             _jsAnalyzer = new JsAnalyzer(limits);
-            if (_analysisLevel != AnalysisLevel.None) {
+            if (this.ShouldAnalyze()) {
```

Comment:
Rather than ShouldAnalyze(), use ShouldEnqueue(), which is more precise. Also don't use ```this.``` to be consistent with the rest of the code here.